tests/test.h

```diff
@@ -81,9 +81,11 @@ extern int test_idempotent_producer;
 
 extern mtx_t test_mtx;
 
-#define TEST_LOCK()   mtx_lock(&test_mtx)
-#define TEST_UNLOCK() mtx_unlock(&test_mtx)
-
+#define TEST_LOCK()                     mtx_lock(&test_mtx)
+#define TEST_UNLOCK()                   mtx_unlock(&test_mtx)
+#define TEST_FIXTURES_FOLDER            "./fixtures"
+#define TEST_FIXTURES_KEYSTORE_PASSWORD "confluent"
```

Comment:
Does this really need to go in test.h if it is only used by one test?

Also, please avoid using "confluent" if possible. librdkafka is not a confluent project.